tests/console/zypper_info.pm

```diff
@@ -29,29 +39,35 @@ sub run() {
     my $expected_package_name_vim = 'Name *: vim';
     die "Missing package name. Expected: /$expected_package_name_vim/"
       unless $info_output_vim =~ /$expected_package_name_vim/;
+}
 
-    if (sle_version_at_least('12-SP3')) {
+sub run() {
+    select_console 'root-console';
+
+    # preparation of source repositories
+    my $cmd;
+    if (check_var('DISTRI', 'sle')) {
+        # SLE maintenance tests are assumed to be SCC registered
+        # and source repositories disabled by default
+        if (get_var('FLAVOR') =~ /-Updates$/) {
+            $cmd = q{mr -e $(zypper -n lr | awk '/-Source/ {print $1}')};
+        }
         # use dvd2 as the src-repository
-        zypper_call('ar --disable --type plaindir cd:///?devices=/dev/sr1 repo-source');
+        else {
+            $cmd = 'ar --type plaindir cd:///?devices=/dev/sr1 repo-source';
+        }
     }
-
-    if (check_var('DISTRI', 'opensuse') || sle_version_at_least('12-SP3')) {
-        test_dvd_src_as_repo();
+    # source repository is disabled by default
+    elsif (check_var('DISTRI', 'opensuse')) {
+        $cmd = "mr -e repo-source";
     }
-}
 
-sub test_dvd_src_as_repo {
-    zypper_call('mr -e repo-source');
-    zypper_call('ref');
-    my $info_output_coreutils = script_output 'zypper info srcpackage:coreutils';
+    zypper_call($cmd);
+    zypper_call("ref");
 
-    my $expected_header_coreutils = 'Information for srcpackage coreutils:';
-    die "Missing info header. Expected: /$expected_header_coreutils/"
-      unless $info_output_coreutils =~ /$expected_header_coreutils/;
-
-    my $expected_package_name_coreutils = 'Name *: coreutils';
-    die "Missing package name. Expected: /$expected_package_name_coreutils/"
-      unless $info_output_coreutils =~ /$expected_package_name_coreutils/;
+    # check for zypper info
+    test_standard_output;
+    test_srcpackage_output if (sle_version_at_least('12-SP2') || check_var('DISTRI', 'opensuse'));
```

Comment:
@mimi1vx, aha, I didn't know that SLE12SP2 couldn't. I thought that only the text output was different SP3. My mistake. Then it is really a feature at SP3